FabricExample/e2e/examplesTests/tests.e2e.ts

```diff
@@ -0,0 +1,111 @@
+import { device, expect, element, by } from 'detox';
+
+describe('Test432', () => {
+  beforeAll(async () => {
+    await device.reloadReactNative();
+  });
+
+  it('should Test432 exist', async () => {
+    await waitFor(element(by.id('root-screen-tests-Test432')))
+    .toBeVisible()
+    .whileElement(by.id('root-screen-examples-scrollview'))
+    .scroll(600, 'down', NaN, 0.85);
+
+    await expect(element(by.id('root-screen-tests-Test432'))).toBeVisible();
+    await element(by.id('root-screen-tests-Test432')).tap();
+  });
+
+  it('home-square should be fully visible', async () => {
```

Comment:
```suggestion
  it('headerRight should be fully visible', async () => {
```

I think we should change naming convention to be more role-specific. This is something I'm coming up as I write this message, so I'm very open for discussion. 

What I mean is - instead of `home-square` we should describe the role of the element in contention (is that a good word here? 😄). For example: `headerRight element should be fully visible`. That makes it much easier to reason about what goes on in the test only by skimming its description.

Reply:
Agreed, it makes sense.

Reply:
2ed530f9171e919d9ad3722006170f54d0977fd7